src/pages/api/properties/[id].js

```diff
@@ -0,0 +1,12 @@
+import { getProperty } from '../../../utils/service-api-client/repairs'
+
+export default async function endpoint(req, res) {
```

Comment:
same comment as above

Reply:
Thanks. [Updated](https://github.com/LBHackney-IT/repairs-hub-frontend/pull/26/commits/35b2262c25a2b8c62917330d05a31eeec5f9fcfa#diff-878483077f770b4fd8cf58fcd922796141740d9db395bbb0424d24b7ba955a34R3)